src/GPL-2.0+.xml

```diff
@@ -280,17 +268,19 @@
          <p>To do so, attach the following notices to the program. It is safest to attach them to the start of each
          source file to most effectively convey the exclusion of warranty; and each file should have at least
          the "copyright" line and a pointer to where the full notice is found.</p>
-         <p><optional>&lt;</optional>one line to give the program's name and an idea of what it does.<optional>&gt;</optional>
-         <br/>Copyright (C) <optional>&lt;</optional>yyyy<optional>&gt;</optional> <optional>&lt;</optional>name of author<optional>&gt;</optional></p>
+         <standardLicenseHeader>
+         <p><alt name="description" match=".+">&lt;one line to give the program's name and an idea of what it does.&gt;</alt><br/>
+         Copyright (C) <alt name="copyright" match=".+">&lt;yyyy&gt; &lt;name of author&gt;</alt></p>
          <p>This program is free software; you can redistribute it and/or modify it under the terms of the GNU
          General Public License as published by the Free Software Foundation; either version 2 of the License,
          or (at your option) any later version.</p>
          <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
          the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
          Public License for more details.</p>
          <p>You should have received a copy of the GNU General Public License along with this program; if not, write
-         to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301<optional>,</optional> USA.
-         Also add information on how to contact you by electronic and paper mail.</p>
```

Comment:
This was missing a `<p>` marker to split “Also add information…” into its own paragraph.  This also affected `GPL-2.0-or-later`, `LGPL-2.1+`, and `LGPL-2.1-or-later`.